lib/table_saw/queries/prepared_insert_statement.rb

```diff
@@ -31,7 +31,7 @@ def column_types
       end
 
       def column_names
-        TableSaw.schema_cache.columns(table_name).map(&:name).join(', ')
+        TableSaw.schema_cache.columns(table_name).map(&:name).map { |name| %Q("#{name}") }.join(', ')
```

Comment:
There's an existing example of quoting columns in `TableSaw::Formats::Copy#quoted_columns`